cudamapper/src/application_parameters.hpp

```diff
@@ -48,6 +48,7 @@ class ApplicationParameters
     int32_t index_size                      = 30;    // i
     int32_t target_index_size               = 30;    // t
     double filtering_parameter              = 1e-5;  // F
+    bool custom_filtering_parameter         = false; // Enables override of the adaptive default for -F (i.e., -F can be set even on small readsets)
```

Comment:
I don't understand the point of this variable, it's not settable by the user so what does it do?

Reply:
Ah, I should have just placed this one in the `.cpp`.

Reply:
I realized that since this is needed by multiple functions in the class, it makes sense to have it be a class member. I've made it private to reflect that users should not modify it directly.

Reply:
Ok, I now see this discussion as well. I think this value should be declared in constructor and passed around as needed, no need to keep it for the entire lifetime of the object